msprime/simulations.py

```diff
@@ -804,6 +805,24 @@ def read_hapmap(cls, filename):
             f.close()
         return cls(positions, rates)
 
+    def get_mean_recombination_rate(self):
+        """
+        Use numpy to compute the weighted mean recombination rate
```

Comment:
The documentation for a method should tell you *what* it does, not *how* it does it. "Returns the..."